### PR TITLE
[zh-cn] translate contentvisibilityautostatechange in zh-cn

### PR DESCRIPTION
**Pull request by LockingReal**
Provide Chinese translation for 【contentvisibilityautostatechange】this api has updated

### Related issues and pull requests 

https://github.com/mdn/content/pull/24263

@yin1999 🥰

### CODE REVIEW

**Comment by LockingReal** [comment]
I also want to take this opportunity to better understand the process and details of mdn contribution translation so as to make more contributions in the future ~~😝🦾

**Comment by yin1999** [comment]
> Provide Chinese translation for 【contentvisibilityautostatechange】this api has updated

Please pin me after the en-US updates are applied :) thanks for your contribution

**Comment by LockingReal** [comment]
> > Provide Chinese translation for 【contentvisibilityautostatechange】this api has updated
> 
> Please pin me after the en-US updates are applied :) thanks for your contribution

ok~~🍻

File: files/zh-cn/web/api/contentvisibilityautostatechangeevent/index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: ContentVisibilityAutoStateChangeEvent
+slug: Web/API/ContentVisibilityAutoStateChangeEvent
+page-type: web-api-interface
+tags:
+  - Interface
+  - Reference
+  - Experimental
+browser-compat: api.ContentVisibilityAutoStateChangeEvent
```
Comment (yin1999) [inline files/zh-cn/web/api/contentvisibilityautostatechangeevent/index.md:9]: 请移除：

```suggestion
```

File: files/zh-cn/web/api/contentvisibilityautostatechangeevent/index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+---
+
+{{APIRef("CSS Containment")}}{{SeeCompatTable}}
+
+**`ContentVisibilityAutoStateChangeEvent`** 实现了 [CSS Containment Module Level 2](https://www.w3.org/TR/css-contain-2/#content-visibility-auto-state-changed) {{domxref("element/contentvisibilityautostatechanged_event","contentvisibilityautostatechanged")}} event 接口， 可以适用于任何使用了 {{cssxref("content-visibility", "content-visibility: auto")}} 的节点
+
+如果使用了`content-visibility: auto`[relevant to the
+user](https://www.w3.org/TR/css-contain-2/#relevant-to-the-user)，当元素的呈现状态由于任何使元素与用户相关的属性而发生变化时，该事件将在元素上触发。
```
Comment (yin1999) [inline files/zh-cn/web/api/contentvisibilityautostatechangeevent/index.md:17]: 这里的翻译还需修改

File: files/zh-cn/web/api/contentvisibilityautostatechangeevent/index.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- The {{domxref("element/contentvisibilityautostatechange_event", "contentvisibilityautostatechange")}} event
+- [CSS Containment](/zh-CN/docs/Web/CSS/CSS_Containment)
+- The {{cssxref("content-visibility")}} property
+- The {{cssxref("contain")}} property
```
Comment (yin1999) [inline files/zh-cn/web/api/contentvisibilityautostatechangeevent/index.md:72]: 请翻译

File: files/zh-cn/web/api/contentvisibilityautostatechangeevent/index.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+重要的是要注意，由于content-visibility: auto子树元素在语义上保持相关，因此此类子树中的更新仍应继续发生，但允许以降低的优先级发生。这确保了诸如在页面中查找和辅助技术等功能可以访问合理更新的内容。
+
+**Note:** 在过去的提案中我们曾使用 `contentvisibilityautostatechanged`为命名，但在最新的提案中，我们将其更名为 `contentvisibilityautostatechange`
+
+{{InheritanceDiagram}}
+
+## 构造函数
+
+- {{domxref("ContentVisibilityAutoStateChangeEvent.ContentVisibilityAutoStateChangeEvent", "ContentVisibilityAutoStateChangeEvent()")}} {{Experimental_Inline}}
+  - : 构造一个 `ContentVisibilityAutoStateChangeEvent` 实例e.
```
Comment (yin1999) [inline files/zh-cn/web/api/contentvisibilityautostatechangeevent/index.md:30]: 注意标点

File: files/zh-cn/web/api/contentvisibilityautostatechangeevent/index.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    startCanvasUpdates(canvasElem);
+  }
+}
+
+// Call this when the canvas updates need to start.
+function startCanvasUpdates(canvas) {
+  // …
+}
+
+// Call this when the canvas updates need to stop.
```
Comment (yin1999) [inline files/zh-cn/web/api/contentvisibilityautostatechangeevent/index.md:53]: 可以的话，请翻译注释

**Review by yin1999 — CHANGES_REQUESTED** [review]
请对照英文文档更新内容，https://github.com/mdn/content/blob/main/files/en-us/web/api/contentvisibilityautostatechangeevent/index.md

同时，请阅读[简体中文翻译指南](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md)更正常见的问题

File: files/zh-cn/web/api/contentvisibilityautostatechangeevent/index.md (hunk 5 above)
Comment (LockingReal) [reply files/zh-cn/web/api/contentvisibilityautostatechangeevent/index.md:53]: ok，感谢老哥的review，我这将逐一进行更改😁

**Comment by LockingReal** [comment]
@yin1999 老哥很奇怪，我好像rebase了一下main，提交就成这样了，你再看看嘞，又在web/api/element下补充了一个

**Comment by yin1999** [comment]
> @yin1999 老哥很奇怪，我好像rebase了一下main，提交就成这样了，你再看看嘞，又在web/api/element下补充了一个

若没有冲突，请不要使用 rebase。有冲突需要解决的话，用 merge 命令就行

**Pull request closed without merging**